tests/transforms/test_add_grid_transform.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import torch
+from PIL import Image as PILImageModule
+from PIL.Image import Image as PILImage
+from torch.testing import assert_close
+
+from lightly.utils.dependency import torchvision_transforms_v2_available
+
+if not torchvision_transforms_v2_available():
+    pytest.skip("torchvision.transforms.v2 not available", allow_module_level=True)
+from torchvision.tv_tensors import BoundingBoxes, Mask
+
+from lightly.transforms import AddGridTransform
+
+
+# ignore typing due to Any type used in torchvison.transforms.v2.Transform
+@pytest.fixture
+def img_orig() -> PILImage:  # type: ignore[misc]
+    img = PILImageModule.new("RGB", (7, 5))
+    return img
+
+
+@pytest.fixture
+def cols() -> int:
+    return 3
+
+
+@pytest.fixture
+def rows() -> int:
+    return 2
+
+
+def test_AddGridTransform_bbox(rows: int, cols: int) -> None:
+    img_h = 7
+    img_w = 5
+
+    bbox_expected = torch.tensor(
+        [
+            [0.00, 0.00, 1.66, 3.50],
+            [0.00, 3.50, 1.66, 7.00],
+            [1.66, 0.00, 3.33, 3.50],
+            [1.66, 3.50, 3.33, 7.00],
+            [3.33, 0.00, 5.00, 3.50],
+            [3.33, 3.50, 5.00, 7.00],
+        ]
+    )
+
+    tr = AddGridTransform(rows, cols)
+    bbox_empty = BoundingBoxes(
+        torch.zeros(1, 4), format="XYXY", canvas_size=(img_h, img_w)
+    )
+    bbox_tr = tr(bbox_empty)
+    assert_close(bbox_expected, bbox_tr, atol=0.01, rtol=0.01)
+
+
+def test_AddGridTransform_mask(cols: int, rows: int) -> None:
```

Comment:
Very minor: The order of `cols` and `rows` argument is inconsistent between tests.

Reply:
fixed in https://github.com/lightly-ai/lightly/pull/1731/commits/ccaa553d6e42d5ac1d6fdce29eac068e37f9f892